src/lib/seam/components/AccessCodeDetails/AccessCodeDetails.tsx

```diff
@@ -41,18 +42,56 @@ export function AccessCodeDetails({
     )
   }
 
+  const filterCodeErrors = () => {
+    return (
+      accessCode?.errors?.filter((error) => {
```

Comment:
`accessCode.errors` should always return an array.
@razor-x is fixing that in the types:
https://hello-seam.slack.com/archives/C01D0J9ACLF/p1687914492628269?thread_ts=1687906591.227049&cid=C01D0J9ACLF

Reply:
I presume this applies to `warnings` as well. If not, I can revert the changes from [ea05f15](https://github.com/seamapi/react/pull/243/commits/ea05f15d80a87a102afc00bfd9b7321003a345da)

Reply:
Naw, for warnings, we don't merge device and account warnings, so we just have to show the available warnings

Reply:
I meant for `warnings` always being an array. It should be, yeah?